test/e2e.sh

```diff
@@ -0,0 +1,95 @@
+#! /bin/bash
+
+# Get running RHCOS version
+find_rhcos_version() {
+    worker_node=$(kubectl get nodes --selector='node-role.kubernetes.io/worker' -ojsonpath={.items[].metadata.name})
```

Comment:
I think we need to quote the JSON path here?
```suggestion
    worker_node=$(kubectl get nodes --selector='node-role.kubernetes.io/worker' -ojsonpath='{.items[].metadata.name}')
```

Reply:
I don't think so,
```
bash-5.1$ echo {}
{}
bash-5.1$ echo {[]
{[]
bash-5.1$ ^C
bash-5.1$ echo {}
{}
bash-5.1$ echo []
[]
bash-5.1$ echo ()
> (waiting for user input)
```
I think as long as there's no space, `$`, `()`, quoting isn't required

Reply:
I added the quotes, at the least it looks more consistent with the --selector option before it.